third_party/terraform/resources/resource_container_cluster.go.erb

```diff
@@ -784,6 +787,55 @@ func resourceContainerCluster() *schema.Resource {
 	}
 }
 
+// Setting 0 for guest accelerator count has the effect of removing all of the guest_accelerators. Since the API
+// returns an empty block we are removing any blocks where count=0 from the comparison between the two states.
```

Comment:
This is untrue, right? We're just blanking out the diff if there's a mismatch

Reply:
Right, forgot to update this comment